lib/product-import.coffee

```diff
@@ -161,39 +172,48 @@ class ProductImport
         if existingProduct?
           @_fetchSameForAllAttributesOfProductType(prodToProcess.productType)
           .then (sameForAllAttributes) =>
-            @_validateEnums(prodToProcess, @_cache.productType[prodToProcess.productType.id])
-            .then (updateActions) =>
-              if updateActions and _.size(updateActions.actions) > 0 then @_updateEnumUpdateActions(enumUpdateActions, updateActions)
-              @_prepareUpdateProduct(prodToProcess, existingProduct).then (preparedProduct) =>
-                synced = @sync.buildActions(preparedProduct, existingProduct, sameForAllAttributes)
-                if synced.shouldUpdate()
-                  @client.products.byId(synced.getUpdateId()).update(synced.getUpdatePayload())
-                else
-                  Promise.resolve statusCode: 304
+            @_prepareUpdateProduct(prodToProcess, existingProduct).then (preparedProduct) =>
+              synced = @sync.buildActions(preparedProduct, existingProduct, sameForAllAttributes)
+              if synced.shouldUpdate()
+                @client.products.byId(synced.getUpdateId()).update(synced.getUpdatePayload())
+              else
+                Promise.resolve statusCode: 304
```

Comment:
@emmenko 
need help with that... will do it in the separate refactoring ticket.
https://github.com/sphereio/sphere-product-import/issues/43